frontend/src/scenes/dashboard/newDashboardLogic.ts

```diff
@@ -160,6 +173,7 @@ export const newDashboardLogic = kea<newDashboardLogicType>([
                     lemonToast.error(`Could not create dashboard: ${message}`)
                 }
             }
+            actions.setIsLoading(false)
```

Comment:
when is this set to `true` in this context?

Reply:
It's set to true in the click handler